src/activities/Elsa.Activities.Sql/Contracts/ISqlConnectionStringHandler.cs

```diff
@@ -0,0 +1,9 @@
+using System.Threading.Tasks;
+
+namespace Elsa.Activities.Sql.Contracts
```

Comment:
I had  to touch common Secrets logic which also involves SQL handling. There are a few limitations of Secrets logic currently:
1. all secrets are visible on form as plain text and are loaded for that UI hint
2. all secrets are copied as values whenever they are used - i.e. if you change a secret in Credentials Manager then you will have to edit and republish all workflows that are using it (and you have to remember where you used them)

I agree this could have been a separate PR for that improvement.